CONTRIBUTING.md

```diff
@@ -0,0 +1,512 @@
+# Contributing to opentelemetry-go-build-tools
+
+The Go special interest group (SIG) meets regularly. See the
```

Comment:
We do not have any dedicated SIG for Go-build tools. I think what we have in `README.md` is good enough.